auth/data.js

```diff
@@ -22,14 +33,24 @@ var Client = Entity.configure({
      * - lastRotated      // Last time accessToken was reset
      * (more properties may be added in the future)
      */
-    details:        Entity.types.JSON
+    details:        Entity.types.JSON,
+    scopes:         Entity.types.JSON,  // new in v2
+    disabled:       Entity.types.Number // new in v2
```

Comment:
`JSON` is probably the best way to serialize a boolean... unless of course you want to be able to use it in a query...

Anyways, sad we don't have boolean in `azure-entities`, it's kind of bad to miss that :)